src/test/java/com/wikia/webdriver/testcases/portableinfoboxtests/PortableInfoboxTests.java

```diff
@@ -95,11 +95,11 @@ public void verifyExternalLinksRedirecting() {
         .open(PageContent.PORTABLE_INFOBOX_01)
         .getExternalLinkRedirectTitle(0);
 
-    String externalUrl = infobox
+    infobox
```

Comment:
Looking above I think it makes sense to use the ArticleContent helper to pre-populate the page with what the test needs (infobox code)